rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -58,7 +128,11 @@ impl LiveSplit {
         run.metadata_mut().set_speedrun_com_variable("New Game Glitch", "Normal");
         let mut timer = LiveSplitTimer::new(run).unwrap();
         timer.set_current_timing_method(TimingMethod::GameTime);
-        LiveSplit { timer, layout }
+        let time = TimeWithFraction {
+            time: Time::with_digits_format(LiveSplitDigitsFormat::SingleDigitSeconds),
+            fraction: Fraction::with_accuracy(LiveSplitAccuracy::Hundredths),
+        };
```

Comment:
A `TimeWithFraction::new(DigitsFormat, Accuracy)` function would make this nicer to read.